main.js

```diff
@@ -0,0 +1,107 @@
+var VideoView = Backbone.View.extend({
+  className: 'current_video',
+
+  template: Handlebars.compile($('#current-video-template').html()),
+  
+  render: function () {
+    this.$el.html(this.template(this.model.toJSON()));
+
+    return this;
+  }
+});
+
+var VideosView = Backbone.View.extend({
+  className: 'video_list',
+
+  template: Handlebars.compile($('#video-list-template').html()),
+  
+  render: function () {
+    this.$el.html(this.template(this.model.toJSON()));
+
+    return this;
+  }
+});
+
+var VideoModel = Backbone.Model.extend({
+  defaults: function () {
+    return {
+      title: '',
+      description: '',
+      videoId: '',
+      image_url: '',
+    }
+  },
+
+  urlRoot: 'https://www.googleapis.com/youtube/v3/search?part=snippet&maxResults=5&q=cats&type=video&videoEmbeddable=true&key=AIzaSyBMMdn63cR3Sby4uyuYiEWH-xltbXvmGZc',
```

Comment:
You want to define a `url` on the videos collection, not a `urlRoot` on the video model. `urlRoot` implies that you'll be making a request for videos, one at a time, by id.

But you will in fact be getting several videos back at one time, which is why you should do this from a collection (since the collection will deal with many videos).